customer_activity_statement/wizard/customer_activity_statement_wizard.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from datetime import date, timedelta
+from odoo import api, fields, models
+
+
+class CustomerActivityStatementWizard(models.TransientModel):
+    """Customer Activity Statement wizard."""
+
+    _name = 'customer.activity.statement.wizard'
+    _description = 'Customer Activity Statement Wizard'
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        default=lambda self: self.env.user.company_id,
+        string='Company'
+    )
+
+    date_start = fields.Date(required=True,
+                             default=fields.Date.to_string(
```

Comment:
@hveficent by default, when you print a statement, it should be from 1st of january, begining of year.

Reply:
@feketemihai we appreciate your feedback. We must reconsider the logic of this module. Thanks. Feel free to collaborate.

Reply:
@mreficent since the report are close to each other, isn't it easier to have a option in wizard for outstanding, if set only take outstanding, otherwise all activity.

Reply:
We will consider it, thanks.

Reply:
@hveficient @mreficent i can do it on monday to consolidate report, plus issue of reconciled.

Reply:
Thanks, but after some consideration, we prefer to do ourselves.